.github/workflows/ci.yml

```diff
@@ -38,25 +38,47 @@ jobs:
           profile: minimal
           toolchain: stable
           override: true
-      - uses: actions-rs/cargo@v1
+      - uses: actions/cache@v2
         with:
-          command: test
-          args: --release --all-features --manifest-path=compact_str/Cargo.toml -- --include-ignored
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key: ${{ github.event.repository.name }}-${{ runner.os }}-cargo-test-v3
+      - name: Install cargo-nextest
+         run: cargo install cargo-nextest
+      - name: cargo test
+        run: |
+          cargo nextest run --release --all-features --manifest-path=compact_str/Cargo.toml -- --include-ignored
```

Comment:
I'd prefer to stick with normal `cargo test` for now as `nextest` can actually cause slowdowns on Windows and macOS. It's a very promising test running suite though!

Reply:
How about using `cargo-nextest` for linux, but continue to use `cargo-test` for windows and macOS?

Reply:
OK, I've reverted to using `cargo-test` for windows and macOS.

Though TBF, I've been using `cargo-nextest` for macOS on M1 and it is much faster to run tests for this crate and `tokio-rs/bytes`.